### ISSUE TITLE
aws_wafregional_xss_match_set crashes terraform

### ISSUE THREAD

**Issue by igorantunes1984** [akamai/terraform-provider-akamai#141]
Hi,

When I try to create a XSS Match Condition on AWS using the aws_wafregional_xss_match_set Terraform always crashes.

### Terraform Version
Terraform v0.12.24
+ provider.aws v2.59.0

### Affected Resource(s)
aws_wafregional_xss_match_set

### Panic Output
2020-04-25T19:00:47.795+0100 [DEBUG] plugin.terraform-provider-aws_v2.59.0_x4: panic: interface conversion: interface {} is *schema.Set, not []interface {}

### Expected Behavior
A XSS Match set with several rules should have been created.

### Actual Behavior
An empty Match Set was created

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Create a tf file to create a new XSS Match Set using the regional WAF (aws_wafregional_xss_match_set) with some rules.
2. Save.
3. From the terminal execute: terraform apply

After this it will crash.

### Important Factoids
Running on MacOS with aws key and aws secret key

**Comment by adriangrimshaw** [comment]
I have observed this exact isssue, using code identical to that applied previously (at that time aws provider v2.31.0).

I notice a slight different in syntax between:
1) `resource "aws_waf_xss_match_set"`  which has  **xss_match_tuples**
2) `resource "aws_wafregional_xss_match_set"`  which has  **xss_match_tuple**

In addition, since this code was last applied, AWS WAFv2 was implemented. All "WAF Classic" should run in parallel, without any API changes, but just mentioning it in case it helps identify the cause.
